distribution/archives/integ-test-zip/src/javaRestTest/java/org/elasticsearch/test/rest/RequestsWithoutContentIT.java

```diff
@@ -25,6 +25,7 @@ public void testIndexMissingBody() throws IOException {
         assertResponseException(responseException, "request body is required");
     }
 
+    @AwaitsFix(bugUrl = "need to decide how to handle this scenario")
```

Comment:
I have this change in my draft of HttpObjectAggregator removal. It should address `no-content` partially, for known length I use content-length header, for chunked there is no way to tell so it's always true.

https://github.com/elastic/elasticsearch/pull/112120/files#diff-b6d89d18f95a49d731741f926ee22d01f0dd8039de82382e68c1862e19f22b04R282-R297